meinberlin/templates/header.html

```diff
@@ -7,21 +7,26 @@
             <a href="/" class="main-nav__logo" rel="home">
                 <img src="{% static 'images/LOGO.png' %}" alt="mein.berlin"/>
             </a>
-            <ul>
-                <li>
-                    <a class="main-nav__item" href="{% url 'project-list' %}">{% trans 'Participation processes' %}</a>
-                </li>
-                {% get_menu "topnav" as topnav %}
-                {% for item in topnav %}
-                    <li>
-                        <a class="main-nav__item" href="{{ item.url }}">{{ item.title }}</a>
+            <button class="button button--bordered main-nav__toggle" data-toggle="collapse" data-target="#menu-collapse" aria-expanded="false" aria-controls="menu-collapse">
+                <i class="fa fa-bars"></i>
```

Comment:
icon should be hidden and there should be a text alternative (e.g. `title` or `aria-label`)